Career-Day/career_day_BE
# Postman Test Integration

# Description
* Add functionality to perform integration tests with Postman
* Update the Readme with new testing and Team
* Fix the bug where when the `db_seed` command is run it does not delete old seeded jobs. Now when you run the command it deletes the old seeds and makes it so newly added seeds begin with the primary key of 1.
* Even though this branch is called `dan_docker` it does not implement any Docker functionality.

# Closes issue(s)
#27 #18 #19 

# How to test / reproduce
The way to test is spelled out in the Readme:
7. run integration tests
    * First, click the button below to download the Career Day Postman collections to your local client:
    [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/c226a3ff3a643f0456f7)
    * Next, ensure the app is running with `flask run` in the command line
    * Finally, to run the tests, in Postman:
      1. Click `Runner` at the top left
      2. In the new window that opens up, click on the Career Day collection on the left
      3. Then click the blue Run Career Day button to run all the tests
    * To examine or run just one test, in the main Postman app click on the `Tests` tab of one of the requests and click the blue `Send` button

# Screenshots

# Changes include

# Checklist

# Other comments
